Gemfile.lock

```diff
@@ -47,77 +47,77 @@ GIT
 
 GIT
   remote: https://github.com/decidim/decidim.git
-  revision: 5d79046024cb74e04c036f4349e4cdd32d04963a
-  branch: release/0.21-stable
+  revision: 7b40c244de405253ff5af3607e0da8c51c0cb4bc
```

Comment:
We should run the command below in order to get the latest backports, since we've recently ported an important new bugfix.
`bundle update --source decidim`